spec/ohloh_scm/git_activity_spec.rb

```diff
@@ -0,0 +1,366 @@
+# frozen_string_literal: true
```

Comment:
Is # frozen_string_literal: true still necessary in later versions of Ruby?

Reply:
It wont be necessary in ruby v4. They were planning to make it obsolete in ruby v3,  but now the plans are delayed. There are memory performance benefits to having this in every file.
https://bugs.ruby-lang.org/issues/11473
https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/